src/pages/api/quran-reflect.ts

```diff
@@ -0,0 +1,15 @@
+import getReflectionData from 'src/api';
```

Comment:
Originally we need `/api` because we need to avoid CORS issue. Can't request directly to quranreflect.com

But since we need to need to get `Verse` data as well. I just call the API directly from here.